target_tracking_node:
  ros__parameters:
    clutter_rate: 0.1
    probability_of_detection: 0.9
    probability_of_survival: 0.99
    gate_threshold: 1.5

    confirmation_threshold: 0.7
    deletion_threshold: 0.1

    std_velocity: 0.2
    std_sensor: 0.5

    update_interval_ms: 600

    world_frame: "world_frame"